src/server/cluster/cluster_family_test.cc

```diff
@@ -733,7 +803,33 @@ TEST_F(ClusterFamilyTest, Keyslot) {
             CheckedInt({"cluster", "keyslot", "123{def}456"}));
 }
 
-class ClusterFamilyEmulatedTest : public BaseFamilyTest {
+TEST_F(ClusterFamilyTest, FlushSlots) {
+  EXPECT_EQ(Run({"debug", "populate", "100", "key", "4", "slots", "0", "1"}), "OK");
+
+  EXPECT_THAT(RunAdmin({"dflycluster", "getslotinfo", "slots", "0", "1"}),
+              RespArray(ElementsAre(RespArray(ElementsAre(IntArg(0), "key_count", Not(IntArg(0)),
+                                                          "total_reads", _, "total_writes", _)),
+                                    RespArray(ElementsAre(IntArg(1), "key_count", Not(IntArg(0)),
+                                                          "total_reads", _, "total_writes", _)))));
+
+  EXPECT_EQ(RunAdmin({"dflycluster", "flushslots", "0"}), "OK");
```

Comment:
flushslots create a fiber that is deleting the data from slots. You are not populating so much data in the test but I think that we cant say for sure that this will finish before the next command you run in the test